shared/params.go

```diff
@@ -120,14 +120,20 @@ func ParseProductSpec(spec string) (productSpec ProductSpec, err error) {
 	return productSpec, nil
 }
 
+// ParseProductSpecUnsafe ignores any potential error parsing the given product spec.
+func ParseProductSpecUnsafe(s string) ProductSpec {
+	parsed, _ := ParseProductSpec(s)
+	return parsed
+}
+
 // ParseProduct parses the `browser-version-os-version` input as a Product struct.
 func ParseProduct(product string) (result Product, err error) {
 	pieces := strings.Split(product, "-")
 	if len(pieces) > 4 {
 		return result, fmt.Errorf("invalid product: %s", product)
 	}
 	result = Product{
-		BrowserName: pieces[0],
+		BrowserName: strings.ToLower(pieces[0]),
```

Comment:
Why is `strings.ToLower` needed? Doesn't the frontend (more specifically ohm) already convert the keywords to lower-case?

Reply:
Because the tests had capitalized names and there's no reason that we shouldn't parse `product=Chrome` as the chrome browser.